plugin/evm/upgrade/acp176/acp176.go

```diff
@@ -19,8 +19,20 @@ import (
 	"github.com/holiman/uint256"
 )
 
+var (
+	// MinTargetPerSecond is a variable so that it can be changed by external Go code, notably
+	// for load testing in the avalanchego repository. You might want to also update the
+	// following variables value since they depend on MinMaxPerSecond:
+	// - MinTargetPerSecond
+	// - MinMaxPerSecond
+	// - MinMaxCapacity
+	// - eth/gasprice.DefaultMinGasUsed
+	MinTargetPerSecond = gas.Price(1_000_000) // P
```

Comment:
I believe the intention is to increase this, right? Can't we accomplish the same thing by increasing the gas target in coreth config?

Reply:
maybe not directly this variable, but we can increase the target gas per second ultimately

Reply:
Having a look, but so far it didn't seem to work. Changing this to be a draft for now